.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Details:
+ - *item1*
+ - *...*
+
+### Tickets:
+ - *ticket-id*
+
+### Connected PRs:
```

Comment:
I think it's not needed. If there is a connected PR, then it can be mentioned in the description.

Reply:
Ok, will remove.

Reply:
Fixed